package-builders/install-rust.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Given that this file is used in multiple builder images, please just create a `scripts` directory in the root of the repo and put it there instead of putting it in one of the other directories.